### PR TITLE
Query params update

### LINKED ISSUE #172: Ajax filter orgnanisation/person failed
The request ajax on people categorie for the filter organisation/person is failing.

### PR DESCRIPTION
**Pull request by SimonLab**
# to be merge after #196 
resolve comment on #171 and resolve #172
@nelsonic do you mind to review this PR, thanks!
The query params object doc can be found on the [search page wiki](https://github.com/TheScienceMuseum/collectionsonline/wiki/Search)

### CODE REVIEW

**Comment by nelsonic** [comment]
👀

**Comment by SimonLab** [comment]
Sorry @nelsonic this PR is not ready yet I'm working on refactoring page.js first #189

**Comment by nelsonic** [comment]
@SimonLab noted. so should I be looking at #181 instead ?

**Comment by SimonLab** [comment]
yes #181 first, thanks

**Comment by nelsonic** [comment]
@SimonLab assign when ready for review. 👍

**Comment by SimonLab** [comment]
@nelsonic @Danwhy  this branch is ready for review

**Comment by nelsonic** [comment]
@SimonLab please merge master into this branch to reduce diff. (_then ping back_) 👍

File: client/lib/get-qs.js [hunk 1]
```diff
@@ -1,27 +1,14 @@
...
-  Object.keys(selected).forEach(el => {
-    qs[el] = Object.keys(selected[el]).join();
-  });
-
-  return qs;
+module.exports = function () {
+  /**
+  * select all the input checkbox checked, all the other input but not the checkbox (already selected) and not fields[type] already on url
+  */
+  var queryParams = $('#search-form input:checked, #search-form input:not([type="checkbox"],[name="fields[type]"]) ').filter(function (i, el) {
```
Comment (nelsonic) [inline client/lib/get-qs.js]: nice. 😉

File: lib/query-params/format-value.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+const splitOnUnescapedCommas = require('../../client/lib/split-commas.js');
```
Comment (nelsonic) [inline lib/query-params/format-value.js]: I can't find a _unit_ test for split-commas.js, am I just not looking in the right place?

Comment (SimonLab) [inline lib/query-params/format-value.js]: Can't find it either. I'm going to add one.

**Comment by nelsonic** [comment]
LGTM

**Pull request merged**